packages/gatsby/src/utils/__tests__/create-content-digest.js

```diff
@@ -0,0 +1,19 @@
+const createContentDigest = require(`../create-content-digest`)
+
+describe(`Create content digest`, () => {
+  it(`returns the content digest`, () => {
+    const content = JSON.stringify({ id: 1 })
+
+    const contentDigest = createContentDigest(content)
+
+    expect(contentDigest).toEqual(`d2ce28b9a7fd7e4407e2b0fd499b7fe4`)
+  })
+
+  it(`throws an error when the data argument is not a string`, () => {
```

Comment:
Could we actually handle non strings args? One less thing that plugin authors need to worry about

Reply:
Yeah just stringify whatever comes in.

Reply:
That's a good point